### PR TITLE
Post bandit security issues within a github pull request

### PR DESCRIPTION
**Pull request by realjanpaulus**

### CODE REVIEW

File: bandit-github/setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+setup(
+    name='bandit-github',
+    version='1.0.0',
+    packages=find_packages(include=['bandit_github']),
+    install_requires=[
+        "bandit",
+        "requests"
+    ],
+    entry_points={'bandit.formatters': ['github = bandit_github.formatter:report']}
+)
```
Comment (severinsimmler) [inline bandit-github/setup.py:13]: Please run `black` on all files and make sure that formatting is consistent (you use both `'` and `"` here)

File: bandit-github/bandit_github/formatter.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    https://bandit.readthedocs.io/en/latest/formatters/index.html#example-formatter
+
+    Parameters
+    ----------
+    manager : _type_
+       A bandit manager object
+    sev_level : _type_
+        Filtering severity level
+    conf_level : _type_
+        Filtering confidence level
```
Comment (severinsimmler) [inline bandit-github/bandit_github/formatter.py:185]: Missing types

File: bandit-github/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Bandit github
+
+Posts [bandit](https://github.com/PyCQA/bandit) security issues within a pull request.
```
Comment (severinsimmler) [inline bandit-github/README.md:3]:
```suggestion
Posts [bandit](https://github.com/PyCQA/bandit) security issues as comment in a pull request.
```

File: bandit-github/bandit_github/formatter.py (hunk 2 above)
Comment (realjanpaulus) [reply bandit-github/bandit_github/formatter.py:185]: By design

File: bandit-github/action.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  ini_path:
+    description: 'path to a .bandit file that supplies command line arguments'
+    required: false
+    default: 'DEFAULT'
+  GITHUB_TOKEN:
+    description: 'Github token of the repository (automatically created by Github)'
+    required: true
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```
Comment (sberz) [inline bandit-github/action.yml:37]: This image takes a while to build. Use a prebuild Image instead:
```suggestion
runs:
  using: 'docker'
  image: 'docker://ghcr.io/snapaddy/actions-bandit-github:1.2.0'
```

Also requires changes in:
- docker-bake.hcl
- .github/workflows/release.yaml

**Review by sberz — APPROVED** [review]

**Pull request merged**